react-table/src/Table.tsx

```diff
@@ -157,6 +161,7 @@ function Cell<T>(props: ICellProps<T>) {
         <td
             style={css}
             onClick={(e) => props.Click({ colKey: props.DataKey, colField: props.DataField, row: props.Object, data: getFieldValue(), index: props.RowIndex }, e)}
+            draggable={props.DragStart !== undefined} onDragStart={props.DragStart}
```

Comment:
it may be worth modeling `onDragStart` like `onClick` with `colKey, colField, row, index` in the parameters for future work